.github/semantic.yml

```diff
@@ -0,0 +1,33 @@
+# Config ref: https://github.com/Ezard/semantic-prs
+
+# Validate the PR title, and ignore all commit messages
+titleOnly: true
+
+# Provides a custom URL for the "Details" link, which appears next to the success/failure message from the app:
+targetUrl: https://github.com/meltano/sdk/blob/main/CONTRIBUTING.md#semantic-pull-requests
+
+# The values allowed for the "type" part of the PR title/commit message.
+# e.g. for a PR title/commit message of "feat: add some stuff", the type would be "feat"
+types:
+  - ci
+  - chore
+  - build
+  - docs
+  - feat
+  - fix
+  - perf
+  - refactor
+  - revert
+  - style
+  - test
+
+# The values allowed for the "scope" part of the PR title/commit message.
+# e.g. for a PR title/commit message of "feat(awesome-feature): add some stuff",
+#      the scope would be "awesome-feature"
+scopes:
+  - sdk       # SDK only - temporary until sdk has its own repo
```

Comment:
This can probably be removed, since you'll be moving the extension SDK into the EDK repo soon after merging this, and we can iterate on it there.

Reply:
Will drop "templates" too since thats gonna go to the edk repo as well.